pkg/apis/operator/v1/types.go

```diff
@@ -145,6 +166,12 @@ type CalicoNetworkSpec struct {
 	// IPv6 addresses will not be auto-detected.
 	// +optional
 	NodeAddressAutodetectionV6 *NodeAddressAutodetection `json:"nodeAddressAutodetectionV6,omitempty"`
+
+	// HostPort specifies if the plugin for HostPort will be configured as part of the CNI configuration.
+	// Default: Enabled
+	// +optional
+	// +kubebuilder:validation:Enum=Enabled,Disabled
+	HostPort *HostPortType `json:"hostPort,omitempty"`
```

Comment:
I think we should call this `HostPorts` with an `s`.

"Enable or disable host ports" is how I'd say it in english.